xapian-letor/api/letor.cc

```diff
@@ -64,8 +64,8 @@ Letor::set_query(const Xapian::Query & query) {
 }
 
 std::vector<Xapian::docid>
-Letor::letor_rank(const Xapian::MSet & mset, Xapian::FeatureList & flist, const char* model_filename) {
-    return internal->letor_rank(mset, flist, model_filename);
+Letor::letor_rank(const Xapian::MSet & mset, const char* model_filename, Xapian::FeatureList & flist) {
```

Comment:
I'm internall calling `Ranker::load_model_from_file(const char* model_filename)` in this method. We could have a similar function in `Letor` as well, so that `letor_rank(*)` doesn't have to do this. Another reason why `Letor` class should be eliminated and be put inside `Ranker`.